src/profile-logic/profile-data.js

```diff
@@ -1347,14 +1347,14 @@ export function getFriendlyThreadName(
           });
           if (contentThreads.length > 1) {
             const index = 1 + contentThreads.indexOf(thread);
-            label = `Content (${index} of ${contentThreads.length})`;
+            label = `Content Process (${index} of ${contentThreads.length})`;
```

Comment:
I think the formatting in the UI doesn't quite fit, as ithe "Content Proecss (1 of 4)" gets close to the edge of its box. I tried a few options using the inspector:

<img width="181" alt="image" src="https://user-images.githubusercontent.com/1588648/48099771-4fa8f280-e1e6-11e8-94f9-ac4fe532e677.png">

I'm not really sure which I prefer. I think `Content Process (1/4)`.

Anyone else have thoughts here @mstange @digitarald @julienw?

Reply:
I didn't modify the counting style.
There's still space empty space for me: 
![image](https://user-images.githubusercontent.com/8499548/48100683-626c0780-e278-11e8-8c7e-68e2d77edd2d.png)

Maybe we should pursue this in a separate issue? (Maybe the column width should adjust as needed?)

Reply:
I like `(3/4)` as well.